src/mount/canonicalize.c

```diff
@@ -154,7 +154,7 @@ canonicalize_dm_name(const char *ptname)
 {
 	FILE	*f;
 	size_t	sz;
-	char	path[256], name[256], *res = NULL;
+	char	path[268], name[256], *res = NULL;
```

Comment:
Without knowing the diagnostic, it's hard for me to comment too usefully here, but if this value is consistently too small it may be worth a dynamic allocation if we know what the right size actually is.